nexus/types/src/external_api/params.rs

```diff
@@ -1252,7 +1269,9 @@ pub struct VpcRouterUpdate {
 pub struct RouterRouteCreate {
     #[serde(flatten)]
     pub identity: IdentityMetadataCreateParams,
+    /// The location that matched packets should be forwarded to.
     pub target: RouteTarget,
+    /// A CIDR block (or named subnet) which this route will apply to.
```

Comment:
Is this accurate? I see a destination can be an IP, IP net, VPC, or subnet.

Reply:
Thanks for looking over this!

It can be any of those, except for VPCs -- that will likely relax in future with VPC peering, but it's not something that makes sense right now. I'll try and rework the wording for this, I maybe want to make it clearer that 'destination' matches on the address in routed packets.

Reply:
Ok, sounds good. I was asking because I saw more stuff on the `RouteDestination` type. But it's fine if there are things that are allowed by the type that are nonetheless rejected by the endpoint logic as invalid, and I see you're doing that nicely [here](https://github.com/oxidecomputer/omicron/pull/5823/files/d747d01467d4be7797a562f0956489d0541c3f72#diff-0017315ba1221fe0d101b6602d359187f4b40e5c3a79b8aa2be94bd85758b3fcR224-R234).